loop/e2e-test/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "loop-e2e-test",
+  "version": "1.0.0",
+  "description": "End-2-end test for Hello Firefox (Loop)",
+  "scripts": {
+    "pocket": "marionette-mocha --host marionette-firefox-host --timeout 60s --runtime /Volumes/FirefoxNightly.app/Contents/MacOS/firefox-bin tests/loop_signup.js",
+    "test": "./run-tests.sh"
+  },
+  "author": "Mozilla",
+  "license": "MPL",
```

Comment:
Sorry, and I know this was probably my screwup, but can we tweak this string to `MPL-2.0` so it's a valid license? Otherwise newer **npm**s will make some noise. Or I can fix in a future PR, not a big deal.